src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -263,8 +264,19 @@ bool layout_optimizer::can_fuse_reorder(program_node& prev, program_node& next,
         ((fmt_prev == format::b_fs_yx_fsv4 && fmt_next == format::bfyx) && (prev_dt == data_types::u8 || prev_dt == data_types::i8))))
         return true;
 
-    if (next.is_type<eltwise>() && prev_simple && next_simple)
+    if (next.is_type<eltwise>() && prev_simple && next_simple) {
+        // Accumulate type of eltwise becomes fp32 so that it prevents accuracy degradation.
```

Comment:
What about revising comment:
`// If accumulate type of eltwise is impacted from fp32 to fp16, do not fuse to avoid accuracy degradation.`

Reply:
Added